scripts/sct_get_centerline.py

```diff
@@ -36,9 +36,10 @@ def get_parser():
                       type_value="multiple_choice",
                       description="Method used for extracting the centerline.\n"
                                   "optic: automatic spinal cord detection method\n"
-                                  "viewer: manual selection a few points followed by interpolation.",
+                                  "viewer: manual selection a few points followed by interpolation\n"
+                                  "fitseg: automatic spinal cord detection method on allready segmented image.",
```

Comment:
```suggestion
                                  "fitseg: fit a regularized centerline on an already-existing cord segmentation. It will interpolate if slices are missing and extrapolate beyond the segmentation boundaries (i.e., every axial slice will exhibit a centerline pixel).",
```